client/shipping-label/views/index.js

```diff
@@ -49,15 +52,49 @@ class ShippingLabelRootView extends Component {
 		this.setState( { showTooltips } );
 	}
 
+	renderPaymentInfo() {
+		const paymentMethod = this.props.shippingLabel.paymentMethod;
+
+		if ( paymentMethod ) {
+			return (
+				<Notice isCompact={ true } showDismiss={ false } className="wcc-metabox-label-payment inline">
```

Comment:
Nitpick: You can omit the `={ true }`, `<Component propName>` is the same as `<Component propName={ true }>`